tests/com/levi/GameTest.java

```diff
@@ -26,7 +26,6 @@ public void game() {
         locFile2 = "/home/levi-ia/Documentos/github/Battleship/src/com/levi/player2.txt";
```

Comment:
Can you use just the relative path, like `Battleship/src/com/levi/player2.txt` instead of the full path in`locFile1` and `locFile2` variables?

Reply:
Test class outdated.
Deleted in commit e06d702.